### PR TITLE
Create SCES-52004_6624A78C

### PR DESCRIPTION
**Pull request by fox140cv**
Add fps unlock patch, tested ok

### CODE REVIEW

File: patches/SCES-52004_6624A78C [hunk 1]
```diff
@@ -0,0 +1,7 @@
+gametitle=Killzone [PAL-M] SCES-52004 6624A78C
+
+[50/60 FPS]
```
Comment (refractionpcsx2) [inline patches/SCES-52004_6624A78C:3]: is it 50 or 60?

Comment (fox140cv) [reply patches/SCES-52004_6624A78C:3]: In both, both when selecting 50hz and when selecting 60hz

Comment (fox140cv) [reply patches/SCES-52004_6624A78C:3]: In both, both when selecting 50hz and when selecting 60hz

**Comment by refractionpcsx2** [comment]
No .pnach extension on the file

**Comment by kamfretoz** [comment]
can you properly add the .pnach file extension to this one?
otherwise PCSX2 wouldn't detect it

**Comment by fox140cv** [comment]
add .pnach thanks

**Comment by refractionpcsx2** [comment]
Thanks :)

**Pull request merged**